test.json

```diff
@@ -1,145 +1,47 @@
 [
-  {
-    "name": "fs-extra",
-    "repo": "https://github.com/jprichardson/node-fs-extra",
-    "description": "fs-extra contains methods that aren't included in the vanilla Node.js fs package. Such as mkdir -p, cp -r, and rm -rf.",
-    "dependents": 3279,
-    "disable": true
```

Comment:
This package is on standard 14.1.0 and it is now passing. Let's keep it in the test suite and not mark it as disabled anymore.

Pro-tip: You can run `npm run test -- --disabled` to run the test suite against repos marked as disabled. If any of them pass, they can be re-enabled.